From bf6bd8554c9f6989de90b472faef308469f6bb29 Mon Sep 17 00:00:00 2001
From: Siddharth Kumar <siddharth.kumar@einfochips.com>
Date: Wed, 8 Jun 2022 15:44:15 +0530
Subject: [PATCH] My first patch

---
 Hello_yocto.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Hello_yocto.c b/Hello_yocto.c
index ac85e65..5cda3f3 100644
--- a/Hello_yocto.c
+++ b/Hello_yocto.c
@@ -55,7 +55,7 @@ int main()
 
     while(1)
     {
-        printf("\nEnter the choice:\n1: For Insertion in first\n2: For insertion in last\n3: For Exit: ");
+        printf("\nEnter the choice:\n1: For Insertion in first\n2: For insertion in last\n3: For Exit: \n");
         scanf("%d",&i);
         if(i==1)
         {
-- 
2.17.1

