From e568b58383bed02868150c9d861d832cf2261373 Mon Sep 17 00:00:00 2001
From: Siddharth Kumar <siddharth.kumar@einfochips.com>
Date: Wed, 8 Jun 2022 16:17:21 +0530
Subject: [PATCH] My second patch

---
 Hello_yocto.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Hello_yocto.c b/Hello_yocto.c
index ac85e65..d52274c 100644
--- a/Hello_yocto.c
+++ b/Hello_yocto.c
@@ -55,7 +55,7 @@ int main()
 
     while(1)
     {
-        printf("\nEnter the choice:\n1: For Insertion in first\n2: For insertion in last\n3: For Exit: ");
+        printf("\nEnter the choice:\n1: For Insertion in first\n2: For insertion in last\n3: For Exit: \n\n");
         scanf("%d",&i);
         if(i==1)
         {
-- 
2.17.1

